tests/flags/flags.go

```diff
@@ -63,6 +63,8 @@ var MigrationNetworkNIC = "eth1"
 
 var DisableCustomSELinuxPolicy bool
 
+var SlirpNetworkBindingPluginImage = "quay.io/kubevirt/network-slirp-binding:20230830_638c60fc8"
```

Comment:
I believe this commit is redundant now the virt-controller contains a default image,